Cubical/HITs/SetTruncation.agda

```diff
@@ -0,0 +1,55 @@
+{-
+
+This file contains:
+
+- Definition of set truncation and its eliminator
+
+It should *not* depend on the Agda standard library
```

Comment:
I think we can remove this comment from the header

Reply:
fixed.